src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutes.scala

```diff
@@ -12,15 +12,16 @@ import akka.http.scaladsl.model.headers._
 import akka.http.scaladsl.server.RouteResult.Complete
 import akka.http.scaladsl.server.directives.{DebuggingDirectives, LogEntry, LoggingMagnet}
 import org.broadinstitute.dsde.workbench.leonardo.model.NotebookClusterActions.ConnectToCluster
+import org.broadinstitute.dsde.workbench.leonardo.util.RouteHelper
 import org.broadinstitute.dsde.workbench.model.UserInfo
 
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
 
-trait ProxyRoutes extends UserInfoDirectives with CorsSupport { self: LazyLogging =>
+trait ProxyRoutes extends UserInfoDirectives with CorsSupport with RouteHelper { self: LazyLogging =>
   val proxyService: ProxyService
   implicit val executionContext: ExecutionContext
 
-  protected val tokenCookieName = "LeoToken"
+  private val tokenCookieName = "LeoToken"
```

Comment:
Could we define `tokenCookieName` in `RouteHelper` so it's not duplicated in `LeoRoutes` and `ProxyRoutes`?

I feel like it'd be easy to change it in one place and forget the other.